eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -1,5 +1,9 @@
 #Requires -Version 7.0
 $CI_YAML_FILE = "ci.yml"
+
+#mgmt: resourceProvider to sdk package name map
+$packageNameHash = [ordered]@{"vmware" = "avs"; "azure-kusto"="kusto"; "cosmos-db"="cosmosdb";"customer-insights"="customerinsights";"monitor"="insights";"msi"="managedserviceidentity";"web"="appservice"}
```

Comment:
insights package is removed and the monitor package is created, so you can remove `"monitor"="insights"`.

This could be a many to many mapping. For instance the swaggers in Azure.ResourceManager comes from common-type, resources and managementgroups. The resources swaggers will affect Azure.ResourceManager and Azure.ResourceManager.Resources packages.

Reply:
Why are we hardcoding these here? Should this info be part of the configuration instead?

Reply:
If we add this info in configuration such as readme.md, we need to parse this information out at first.

Reply:
Aren't you already reading some of that configuration via Get-ResourceProviderFromReadme? If possible we should try to avoid hardcoding these extra configurations because they become difficult to maintain and leak details into every place that needs this information.

Reply:
Hi @weshaggard  Usually the service-directory has the same name of the folder under azure-rest-api-specs/specification, so we can this information via Get-ResourceProviderFromReadme. But in .NET management-plane, there are some exceptions that it will not use different name as sdk serivce-directory, such as `vmware`. In azure-rest-api-specs it called `wmware` , but the service directory for managment-plane sdk is 'avs'. This mapping is to handle such exceptions.

Reply:
If this an exceptional list that we don't expect to be added to and we aren't repeating this same lookup logic somewhere else than I think we could live with this. However, please add a note in the comment about why this exists.